app/controllers/concerns/two_factor_authenticatable.rb

```diff
@@ -6,6 +6,8 @@ module TwoFactorAuthenticatable
     before_action :handle_two_factor_authentication
     before_action :check_already_authenticated
     before_action :reset_attempt_count_if_user_no_longer_locked_out, only: :create
+
+    before_action :apply_secure_headers_override, only: :show
   end
```

Comment:
it seems weird to me that we would put an Openid-specific method/logic in a concern shared by non-Openid controllers. Should this maybe go in the consuming controller rather than the concern?